CommunicationServer/Application/Entities/ChessTimer.cs

```diff
@@ -71,7 +71,7 @@ private void OnOneSecElapsed(object? sender, System.Timers.ElapsedEventArgs e)
             StopTimers();
             ThrowEvent(this, EventArgs.Empty, new JoinedGameStreamDto()
             {
```

Comment:
So there is a funny bug where you join to two games and you get the timer updated to both of them. I think at least disabling join button after joining would be a good idea